go/src/koding/klientctl/main.go

```diff
@@ -743,6 +744,26 @@ error opening: %s
 					},
 				},
 			}, {
+				Name:        "sync",
+				Usage:       "Manage mount synchronization.",
+				Description: cmdDescriptions["mount-sync"],
+				Action:      ctlcli.ExitErrAction(MachineSyncMount, log, "sync"),
+				Flags: []cli.Flag{
+					cli.DurationFlag{
+						Name:  "timeout, t",
+						Usage: "Maximum time to wait.",
+						Value: time.Minute,
+					},
+					cli.BoolFlag{
+						Name:  "pause",
```

Comment:
Initially, my goal was to do like you wrote. However, there are two issues with this approach:

1) Subcommand `pause`/`resume` and directory `pause`/`resume` ambiguity:

If we have `./pause` directory, calling `kd mount sync pause` could indicate either we want to pause synchronization or trigger sync on `./pause` directory. This can be solved by some heuristics that takes into account that mounts cannot be nested but it's still confusing. 

2) Subcommand `pause`/`resume` and mount name ambiguity:

If #11068 is done, calling `kd mount sync pause` may be either pausing synchronization for mount in WD or pausing synchronization for mount with name `pause`.

Generally, I think that the subcommand syntax over option one will be very hard to understand for the potential users.